scripts/publish.sh

```diff
@@ -29,6 +28,5 @@ for crate in "${crates[@]}"; do
     .manifest_path')
 
     # Publish to crates.io
-    cargo publish --manifest-path "$crate_manifest" --token "$CRATES_IO_TOKEN"
-    cargo clean --manifest-path "$crate_manifest"
+    cargo publish --manifest-path "$crate_manifest" "$@" || echo "failed"
```

Comment:
Unfortunately `|| echo "failed"` is needed for retrying the publish. 

https://github.com/ChainSafe/fil-actor-states/actions/runs/9583961899/job/26426873424#step:4:436
```
Caused by:
  the remote server responded with an error (status 400 Bad Request): crate version `14.0.0` is already uploaded
Error: Process completed with exit code 101.
```

Reply:
Can we manually do the publish to avoid this `OR`?

Reply:
> Can we manually do the publish to avoid this `OR`?

@LesnyRumcajs Actually yes. Fortunately, `workflow_dispatch:` is configured for the workflow. I'm now retrying the publish in https://github.com/ChainSafe/fil-actor-states/actions/runs/9585248085/job/26430697561 and will remove this `OR` after that.

Reply:
Fixed. Later on we could maybe switch to some more mature solutions

e.g. https://crates.io/crates/publish-action